cf-acceptance-tests/Dockerfile

```diff
@@ -14,16 +14,17 @@ RUN \
  ENV GOPATH /go
  ENV PATH /go/bin:/usr/local/go/bin:$PATH
  RUN \
-  wget https://storage.googleapis.com/golang/go1.7.4.linux-amd64.tar.gz -P /tmp && \
-  tar xzvf /tmp/go1.7.4.linux-amd64.tar.gz -C /usr/local && \
+  wget https://storage.googleapis.com/golang/go1.9.linux-amd64.tar.gz -P /tmp && \
+  tar xzvf /tmp/go1.9.linux-amd64.tar.gz -C /usr/local && \
   mkdir $GOPATH && \
   rm -rf /tmp/*
 
 RUN go get github.com/tools/godep
 RUN go get github.com/onsi/ginkgo/ginkgo
+RUN go get -d github.com/FiloSottile/gvt
```

Comment:
Could you explain (in the commit message) why this was needed?

Reply:
Tis removed.